compel/src/lib/infect.c

```diff
@@ -837,6 +837,45 @@ void compel_relocs_apply(void *mem, void *vbase, size_t size, compel_reloc_t *el
 		} else if (elf_relocs[i].type & COMPEL_TYPE_INT) {
 			int *where = (mem + elf_relocs[i].offset);
 			*where = elf_relocs[i].value + elf_relocs[i].addend + (unsigned long)vbase;
+#ifdef CONFIG_MIPS
+                /*
```

Comment:
It would be nice to have all the MIPS-specific relocs handling in a separate function.